app/main.py

```diff
@@ -1,8 +1,28 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(
+            self,
+            name: str,
+            age: int
+    ) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result = []
+    for person in people:
+        Person(person["name"], person["age"])
+        result.append(Person.people[person["name"]])
```

Comment:
In this implementation it would be better to assign person to some variable and append this variable to result instead of fetching the same person from people class attribute
However it's possible to simplify it using list comprehension, let's do it :)